composer.json

```
@@ -14,7 +14,7 @@
     }
   ],
   "require": {
-    "adyen/php-api-library": "~4.1",
+    "adyen/php-api-library": "~6.0",
```

Also enough to specify it like this
```suggestion
    "adyen/php-api-library": "~6",
```